radixdlt-cli/src/main/java/com/radixdlt/cli/GetMessages.java

```diff
@@ -33,22 +35,24 @@
  * </pre>
  */
 @CommandLine.Command(name = "get-messages", mixinStandardHelpOptions = true,
-        description = "get all the messages")
-class GetMessage implements Runnable {
-
-    @CommandLine.ArgGroup(exclusive = true, multiplicity = "0..1")
-    Composite.IdentityInfo identityInfo
-
-    @Override
-    void run() {
+		description = "Get all the messages")
+public class GetMessages implements Runnable {
+	@CommandLine.ArgGroup(exclusive = true, multiplicity = "0..1")
+	private Composite.IdentityInfo identityInfo;
 
-        RadixApplicationAPI api = Utils.getAPI(identityInfo)
-        println "Retrieving messages..."
-        api.pull()
-        println "Messages:"
-        api.observeMessages().blockingSubscribe(it -> println "  ${it}")
-        println "Done"
-    }
+	@Override
+	public void run() {
+		RadixApplicationAPI api = Utils.getAPI(identityInfo);
+		println("Retrieving messages...");
+		api.pull();
+		println("Messages:");
+		api.observeMessages().blockingSubscribe(it -> println("  " + it.toString()));
+		println("Done");
```

Comment:
I'm not sure that this will ever exit, unless with ctrl-c or similar.

Reply:
OK, then I'll remove last println().